test/unit/test_io.c

```diff
@@ -5,7 +5,9 @@ bool test_r_io_mapsplit (void) {
 	RIO *io = r_io_new ();
 	io->va = true;
 	r_io_open_at (io, "null://2", R_PERM_R, 0LL, UT64_MAX);
+	mu_assert ("UT64_MAX not mapped", r_io_map_is_mapped (io, UT64_MAX));
```

Comment:
you can use `mu_assert_true`  and `mu_assert_false` if you want.
There is also `mu_assert_null` and `mu_assert_notnull`.